MIP104/MIP104.md

```diff
@@ -339,7 +339,7 @@ The Stability Facilitators can trigger executive votes that set up DDMs to Morph
 
 ¤¤¤
 
-Current target exposure of Morpho sUSDe or USDe vault is: 0
+Current target exposure of Morpho sUSDe or USDe vault is: 450,000,000 DAI
```

Comment:
I don't know where this value comes from, so I couldn't verify

Reply:
https://forum.makerdao.com/t/spark-metamorpho-dai-vault-allocation/23987/18